tox.ini

```diff
@@ -34,7 +31,6 @@ commands =
 
 setenv =
     COVERAGE_FILE=.coverage.{envname}
-    plone-5.2-archetypes: BUILDOUT_FILE="plone-5.2.x-archetypes.cfg"
     plone-5.2: BUILDOUT_FILE="plone-5.2.x.cfg"
```

Comment:
The tox.ini needs some love, but tox tests are failing anyway and switching to GHA might be a good idea. 
So I would not bother about it.